packages/block-editor/src/components/link-control/style.scss

```diff
@@ -197,12 +197,14 @@ $preview-image-height: 140px;
 		// of text, particular those with no spaces.
 		// See: https://github.com/WordPress/gutenberg/issues/33586#issuecomment-888921188
 		white-space: pre-wrap;
+		overflow-wrap: break-word;
 		word-wrap: break-word;
+		word-break: break-word;
```

Comment:
Do we need this? It's [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break)

Reply:
We do need `word-break`, yes, that's what fixed the issue. But good call, I moved to `break-all` instead of `break-word` 👍 👍

<img width="901" alt="Screenshot 2021-11-30 at 12 20 29" src="https://user-images.githubusercontent.com/1204802/144038455-aad8476e-85d6-4d09-8162-1678b3ceca3c.png">